conf/application.conf

```diff
@@ -50,4 +50,24 @@ get-address-io-api {
     key = "q0M6MVJQm0CslqDvMcKUDQ6396"
 }
 
+play.filters.headers.frameOptions = "SAMEORIGIN"
+play.filters.headers.permittedCrossDomainPolicies=null
+play.filters.headers.contentSecurityPolicy=null
+
+play.filters.csrf {
+    header {
+        bypassHeaders {
+            X-Requested-With = "*"
+            Csrf-Token = "nocheck"
+        }
+        protectHeaders = null
+    }
+    bypassCorsTrustedOrigins = false
+    method {
+        whiteList = []
+        blackList = ["POST"]
+    }
+    contentType.blackList = ["application/x-www-form-urlencoded", "multipart/form-data", "text/plain"]
```

Comment:
We may want to double-check this...

Reply:
I guess it's the same as membership did to get the "old" behaviour.  Maybe they are planning to do whatever the official fix is in due course.